### PR TITLE
docs: adds google analytics tracking

### PR DESCRIPTION
**Pull request by nilekhc**
Signed-off-by: Nilekh Chaudhari <1626598+nilekhc@users.noreply.github.com>

**What this PR does / why we need it**:

Adds Google Analytics

**Which issue(s) does this PR fix** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by nilekhc** [comment]
cc @maxsmythe @ritazh @sozercan

File: website/docusaurus.config.js
Comment (sozercan) [inline website/docusaurus.config.js:52]: can you add the maintainer to this?

Comment (nilekhc) [reply website/docusaurus.config.js:52]: I have added you. For other maintainers please let me know your id.

Comment (nilekhc) [reply website/docusaurus.config.js:52]: @maxsmythe Yes. I created a GA account where I can add users to grant access. Please let me know if there is any better way to create and manage GA accounts for both GK websites.

**Review by sozercan — APPROVED** [review]

**Review by sozercan — CHANGES_REQUESTED** [review]
/hold @nilekhc is working on additional testing

**Comment by nilekhc** [comment]
> /hold @nilekhc is working on additional testing

@sozercan I added a check for the website build and is ready for review. PTAL.

**Comment by nilekhc** [comment]
> > /hold @nilekhc is working on additional testing
> 
> @sozercan I added a check for the website build and is ready for review. PTAL.

@sozercan It's ready for review. PTAL

**Review by sozercan — APPROVED** [review]

File: website/docusaurus.config.js
Comment (maxsmythe) [reply website/docusaurus.config.js:47]: Don't know much about analytics sadly :/

What kind of ID did you need?

Comment (nilekhc) [reply website/docusaurus.config.js:47]: @maxsmythe Your Gmail id. I can grant you access to that.

Comment (maxsmythe) [reply website/docusaurus.config.js:47]: Oh, perfect! smythe@google.com , LMK if that doesn't work

**Pull request merged**
